ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Must-Support-Flags.md

```diff
@@ -1,13 +1,34 @@
+
+    Hinweis: Bei den Testszenarien von READ-Interaktionen im Rahmen des Bestätigungsverfahrens werden für MS-Elemente Informationen vorgegeben, die in den Systemen erfasst und über die FHIR-Schnittstelle reproduziert werden MÜSSEN, unabhängig von der angegebenen Kardinalität.
 
 2. Im Kontext der Verarbeitung von FHIR-Ressourcen:
 
-    Wenn die entsprechende Information in einer Ressource vorhanden ist, so MUSS diese vom verarbeitenden System ausgewertet, persistiert und dem Anwender dargestellt werden können.
+    Elemente, die mit MS gekennzeichnet sind, MÜSSEN vom empfangenden System verarbeitet werden. Dies bedeutet:
+    - Das System MUSS zumindest in der Lage sein, diese Elemente in der eigenen Persistenz-Ebene zu speichern
+    - Das System MUSS bei einer erneuten Abfrage der Information über die FHIR-Schnittstelle in der Lage sein diese Information zu reproduzieren (dies ggf. hinreichend bei Middleware)
+    - Systeme MÜSSEN die entsprechenden Elemente zur Anzeige bringen können (ausgenommen davon sind Systeme mit reiner Middleware-Funktion, die über keine grafische Nutzeroberfläche verfügen).
+    - Systeme KÖNNEN es darüber hinaus ermöglichen, dass die jeweiligen Informationen vom Anwender ergänzt oder editiert werden. 
+    - MS-Elemente, die als optional gekennzeichnet sind (0..-Kardinalität) KÖNNEN bei der Übermittlung einer Instanz fehlen, wenn die entsprechende Information im sendenden System unbekannt, nicht zutreffend ist oder (noch) nicht erhoben wurde. Das Fehlen optionaler MS-Elemente DARF bei der Verarbeitung NICHT zu einem Fehler führen.
+
+    Hinweis: Bei den Testszenarien von WRITE/UPDATE-Interaktionen im Rahmen des Bestätigungsverfahrens werden Instanzen an das zu testende System übermittelt, in denen alle MS-Elemente, unabhängig von Ihrer Kardinalität, gegeben sind. Diese MÜSSEN von den getesteten Systemen in einer anschließenden READ-Interaktion vollständig über die FHIR-Schnittstelle reproduziert werden können.
 
-    Sollte ein Hersteller den Bedarf an einer Änderung der Must-Support Vorgaben der ISiK Profile haben, MUSS er von den Profilen ableiten, auf denen die ISiK Profile basieren. Angaben dazu finden sich für jedes Datenobjekt im Kapitel Kompatibilität. In diesem Fall MUSS der Hersteller auch dafür sorgen, dass seine abgeleiteten Profile weiterhin kompatibel mit den ISiK Profilen bleiben.
+Sollte ein Hersteller den Bedarf an einer Änderung der Must-Support Vorgaben der ISiK Profile haben, MUSS er von den Profilen ableiten, auf denen die ISiK Profile basieren. Angaben dazu finden sich für jedes Datenobjekt im Kapitel Kompatibilität. In diesem Fall MUSS der Hersteller auch dafür sorgen, dass seine abgeleiteten Profile weiterhin kompatibel mit den ISiK Profilen bleiben.
```

Comment:
Das verstehe ich nicht. Abgeleitete Profile können vorhandene MS-Flags nicht ändern!

Die einzige Form, in der diese Anforderung Sinn macht wäre "Wenn ein Hersteller neben den in ISiK geforderten Elementen die Verarbeitung *weiterer* Elemente unterstützt, so SOLLTE dies durch abgeleitete Profile, in denen die zusätzlichen Elemente ebenfalls als MS gekennzeichnet sind, dokumentiert und im Rahmen der Schnittstellendokumentation publiziert werden."

Ich denke das MUSS MUSS hier abgeschwächt werden, oder habt ihr vor das zu überprüfen!?

Reply:
@simoneOnFhir Diese Festlegung habe ich nicht modifiziert (bis auf das Leerzeichen). 
Meinetwegen gerne von "MUSS" auf "soll". 
@alexey-tschudnowsky , @alexzautke , was sagt ihr?

Reply:
Naja, wenn MUSS, dann MÜSSEN wir irgendwo Guidance haben, wie ein Hersteller das konkret machen soll. Simplifier? IG Publisher? Muss er ein Package publizieren oder reicht HTML? Wo muss das publiziert werden? Wer prüft das? Was, wenn er ein Element vom Typ CodebleConcept mit einem example Binding aus MS setzt, muss er das dann auf required constrainen und sein eigenes VS publizieren, oder reicht's da einfach ein MS drauf zu klatschen? Reichen Profile oder müssen da auch Beispieldaten dazu? Muss das validieren? Wer validiert das? etc. pp.

Außerdem macht es keinen Sinn, die Forderung nach einer formalen Schnittstellendokumentation nur dann zu stellen, wenn ein Hersteller zusätzliche _Felder_ implementiert. Das gleiche müsste ja auch gelten für die Implementierung zusätzlicher Ressourcentypen, Suchparameter, Operations...
(zumal das als Strafe für die Fleißigen aufgefasst werden könnte und Hersteller entmutigt, mehr zu machen als das notwenige).

Also entweder machen wir das zu einer generellen Anforderung für ALLE, eine formale Schnittstellenbeschreibung abzuliefern, oder wir lassen es. Ich finde es halt komisch in einer kleinen Fußnote so ein riesiges Fass aufzumachen....

Reply:
Ich würde diesen Satz aus der überarbeiteten Fassung herauslassen

Reply:
Ich finde den ersten Vorschlag von @simoneOnFhir gut und übernehme ihn. Normativ läuft es auf eine Streichung des "MUSS" hinaus, der informative Gehalt bleibt aber bestehen.